apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -412,6 +412,7 @@ function initializeBlocklyWrapper(blocklyInstance) {
     return this.setOutput(isOutput, check);
   };
 
+  // A field's 'value' used to be called its 'title' in older versions of Blockly.
```

Comment:
This isn't quite accurate. Fields used to be referred to as titles so either one could have a value.
I might clarify with something like: 
`// Block fields are referred to as titles in CDO Blockly.`

Reply:
Thanks! Will update.